README.md

```diff
@@ -92,18 +92,20 @@ Inside your `AppDelegate.application(_, handleEventsForBackgroundURLSession:comp
 Once the user has logged into your app, inform Streem that they are logged in:
 
 ```swift
-    Streem.sharedInstance.identify(
```

Comment:
We're keeping this method in the SDK so we should keep it documented here.

Reply:
sounds good to me

Reply:
[Done ✅](https://github.com/streem/streem-sdk-ios/pull/16/commits/3abc0de1635541957e6519a1a2bbbaa04d0d0b52)